src/ripple/app/main/Application.cpp

```diff
@@ -1584,7 +1584,7 @@ ApplicationImp::loadLedgerFromFile (
             ledger = ledger.get()["accountState"];
         }
 
-        if (!ledger.get().isArray ())
+        if (!ledger.get().isArrayorNull ())
```

Comment:
Suggestion: capitalize `or` eg. `isArrayOrNull`.

Reply:
I see my comment no longer applies with your last commit.

Reply:
No, it still applies.  I like it.  I'll capitalize `Or` in both places.  Thanks.